resources/assets/js/annotations/components/annotationCanvas/drawInteractions.vue

```diff
@@ -82,7 +82,7 @@ export default {
                 drawInteraction = undefined;
             }
 
-            if (this.isDrawing) {
+            if (this.isDrawing && this.canAdd) {
```

Comment:
I think `isDrawing` can only be true if `canAdd` is true, too. This is handled in `draw()`. Please check if the additional constraint is required here.

Reply:
Without it the 'select a label' info box is shown to a user with no adding permission, if the short-cut is used.